pkg/api/getambassador.io/v3alpha1/crd_tracingservice.go

```diff
@@ -29,16 +29,20 @@ type TraceSampling struct {
 	Overall *int `json:"overall,omitempty"`
 }
 
+// +kubebuilder:validation:Enum=ENVOY;LIGHTSTEP;B3;TRACE_CONTEXT
```

Comment:
Nit: I know some of the existing enums don't follow this, but I believe k8s style is to use PascalCase.

Reply:
I am following the expected values in the Envoy proto files https://github.com/envoyproxy/envoy/blob/96c317837ab7351f6264d25c6e19e09fae3a6054/api/envoy/config/trace/v3/lightstep.proto#L29